deploy/overlays/stanford-staging/kustomization.yaml

```diff
@@ -19,29 +19,29 @@ nameSuffix: -stanford-staging
 namespace: website
 
 resources:
-- ../../base
```

Comment:
is this intended?

Reply:
For my own learning, are these the same?

```
resources:
- A


resources:
  - A
```

Reply:
Yes, which we used in other files. I get this formatting from vscode linter tool.